packages/cli-platform-android/src/config/getMainActivity.ts

```diff
@@ -0,0 +1,106 @@
+import fs from 'fs';
+import {XMLParser, XMLValidator} from 'fast-xml-parser';
+
+const MAIN_ACTION = 'android.intent.action.MAIN';
+const LAUNCHER = 'android.intent.category.LAUNCHER';
+
+interface Activity {
+  [x: string]: any;
```

Comment:
I think you could replace `any` with  `string | undefined`

Reply:
Unfortunately I can't do it because a value can be an object. See here:
```
{
  'intent-filter': {
    action: { '@_android:name': 'android.intent.action.MAIN' },
    category: { '@_android:name': 'android.intent.category.LAUNCHER' }
  },
  '@_android:name': '.MainActivity',
  '@_android:label': '@string/app_name',
  '@_android:configChanges': 'keyboard|keyboardHidden|orientation|screenLayout|screenSize|smallestScreenSize|uiMode',
  '@_android:launchMode': 'singleTask',
  '@_android:windowSoftInputMode': 'adjustResize',
  '@_android:exported': 'true'
}
```